pythoncalculator/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .add import add 
+from .multiply import multiply 
 
```

Comment:
Just noticed that the first line shouldn't have been deleted, try this instead 
```suggestion
from .add import add
from .multiply import multiply
```